packages/visual-editor/src/internal/puck/components/ThemeHeader.tsx

```diff
@@ -58,6 +65,15 @@ export const ThemeHeader = (props: ThemeHeaderProps) => {
     if (fieldListTitle) {
       fieldListTitle.style.display = "none";
     }
+    // Disable component selection in the preview
+    const puckPreview =
+      document.querySelector<HTMLIFrameElement>("#preview-frame");
+    if (puckPreview?.contentDocument) {
+      puckPreview.contentDocument.addEventListener("click", (event) => {
```

Comment:
Add an AbortController so this gets cleaned up properly. Idk if it really matters but it's good practice to do it.